src/components/TabulatorComponent.vue

```diff
@@ -44,8 +97,15 @@ export default class TabulatorComponent extends Vue {
 
   @Watch('options', { deep: true })
   private updateOptions() {
+    const vm = this
+    function customMergeFn(objValue: any, srcValue: any, key: string): any | undefined {
```

Comment:
Maybe we can put that function in another file, something like a merge utils file.
The joint file can handle that?